dynawo/sources/Models/CPP/ModelNetwork/DYNModelPhaseTapChanger.h

```diff
@@ -52,55 +63,61 @@ class ModelPhaseTapChanger : public ModelTapChanger {
    * @param locked : is the tap changer locked ?
    * @param tfoClosed : is the transformer connected ?
    */
-  void evalG(double t, double iValue, bool nodeOff, state_g* g, double disable, double locked, bool tfoClosed);
+  virtual void evalG(double t, double iValue, bool nodeOff, state_g* g,
+                     double disable, double locked, bool tfoClosed);
 
   /**
    * @brief  evaluate discrete values
    * @param t time to use during the evaluation
    * @param g: root values
    * @param network : network of the transformer
    * @param disable : is the tap changer disabled ?
-   * @param P1SupP2 : is the active power evaluated at side 1 is superior to the active power evaluated at side 2  ?
+   * @param P1SupP2 : is the active power evaluated at side 1 is superior to the
+   * active power evaluated at side 2  ?
```

Comment:
Not sure that doxygen reacts well to a line change?

Reply:
done. That is right. doxygen did not say nothing, but I did not check the result.